README.md

```diff
@@ -1,8 +1,10 @@
 # ReactOS Project
 
-Current version: __0.4.6__
 ![ReactOS Logo](https://reactos.org/wiki/images/0/02/ReactOS_logo.png)
 
+[![Release 0.4.6](https://img.shields.io/badge/release-v0.4.6-green.svg)](https://sourceforge.net/projects/reactos/files/ReactOS/0.4.6/)
+[![ReactOS Build Environment](https://img.shields.io/badge/RosBE-v2.1.5-blue.svg)](https://sourceforge.net/projects/reactos/files/RosBE-Windows/i386/2.1.5/)
```

Comment:
Don't forget we have two RosBE. One for Unix-like, one for Windows. I would put both or none (I'd rather go for the second one).

Reply:
Having both of them would be nice actually. But then they should be mentioned separately.

Reply:
@HBelusca I did that just now.